### DISCUSSION TITLE
Is relational filter working ?

### DISCUSSION THREAD

**Discussion by tonytoth** [TriPSs/nestjs-query#21] [Q&A]
Hi,
Does relational filter work on this fork ?
e.g:
```
defaultSort: [
        { field: `test1.sortNo`, direction: SortDirection.ASC },
        { field: `test2.sortNo`, direction: SortDirection.ASC },
        { field: `sortNo`, direction: SortDirection.ASC },
    ]
```
I have this issue while trying to use this:
```
GraphQLError: Enum "ResourceSortFields" cannot represent value: "test1.sortNo"
```

P.S: If you would have to do this, how would you solve this ?

**Comment by TriPSs** [comment]
If I understand you correctly, yes this should work.

If it worked on the old one it should also work here.

**Reply by tonytoth** [reply]
It doesn't work on the old one. I was just curios if you know anything about it.

**Reply by TriPSs** [reply]
As far as I know it should work for filtering, I'm not quite sure about sorting.

**Reply by tonytoth** [reply]
I tried and it didn't work. Maybe it would be a nice feature to be implemented.
